exasol_integration_test_docker_environment/test/test_test_environment_docker_runtime.py

```diff
@@ -44,27 +47,11 @@ def setUpClass(cls):
         cls.docker_environment_name = "test_no_runtime_given"
         cls.on_host_docker_environment, cls.google_cloud_docker_environment = \
             cls.test_environment.spawn_docker_test_environment(cls.docker_environment_name)
-        cls.client = docker.from_env()
-        cls.default_docker_runtime = get_default_docker_runtime(cls)
+        cls.default_docker_runtime = get_default_docker_runtime()
 
     @classmethod
     def tearDownClass(cls):
-        try:
-            cls.test_environment.close()
-        except Exception as e:
-            print(e)
-        try:
-            cls.on_host_docker_environment.close()
-        except Exception as e:
-            print(e)
-        try:
-            cls.google_cloud_docker_environment.close()
-        except Exception as e:
-            print(e)
-        try:
-            cls.client.close()
-        except Exception as e:
-            print(e)
+        close_environments(cls.test_environment, cls.on_host_docker_environment, cls.google_cloud_docker_environment)
```

Comment:
```suggestion
        close_environments(cls.on_host_docker_environment, cls.google_cloud_docker_environment, cls.test_environment)
```

first the docker environment and then the test environment, because the docker environment are port of the test environment